.vscode/settings.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
This is a file with settings for your local code editor and shouldn't be included in the pull request.

Reply:
Yeah, I would add this to the `.gitignore` as well.

Reply:
@naamaarkin you still need to remove this.